docker/Dockerfile

```diff
@@ -40,7 +41,8 @@ RUN apt-get update && apt-get install -y \
     npm install -g cspell@latest && \
     mkdir /opt/cppcheck && git clone $CPPCHECK_REPO --depth 1 --branch $CPPCHECK_VERSION /opt/cppcheck && make -C /opt/cppcheck FILESDIR=/usr/share/cppcheck && make -C /opt/cppcheck install FILESDIR=/usr/share/cppcheck  && \
     mkdir /opt/aarch64-toolchain && curl $AARCH64_TOOLCHAIN_LINK | tar xJ -C /opt/aarch64-toolchain --strip-components=1 && \
-    mkdir /opt/riscv-toolchain && curl $RISCV_TOOLCHAIN_LINK | tar xz -C /opt/riscv-toolchain --strip-components=1
+    mkdir /opt/riscv-toolchain && curl $RISCV_TOOLCHAIN_LINK | tar xz -C /opt/riscv-toolchain --strip-components=1 && \
+    mkdir /opt/asmfmt && wget -c $ASMFMT_LINK -O - | tar xz -C /opt/asmfmt
```

Comment:
Is there anything missing before the | ?

Reply:
This is is basically a bash alias for '/dev/stdout'. Meaning, the output of wget is being written to stdout to pipe it directly to tar.

@ninolomata I'd prefer we keep it consistent and use curl as is done for the toolchains. Later we can remove wget from the installation list altogether.

Reply:
I think the '-' is just a bash alis for /dev/stdout. That is, the command is outputting the downloaded content to stdout which is  then directly piped to tar.